nextjs-app/components/pages/main/Testimonial/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./Testimonial";
```

Comment:
I suppose that we can only create `Testimonial.tsx` and don't need to create a folder and `index.ts` here